apps/mobile/src/components/register-adoption-form/compose/form-steps/pet-upload-photos-step/pet-upload-photos-step.component.tsx

```diff
@@ -0,0 +1,76 @@
+import i18next from 'i18next';
+import { Center, Container, Image, Pressable, View } from 'native-base';
+import React, { useEffect } from 'react';
+import Icon from 'react-native-vector-icons/AntDesign';
+
+import { usePetPhotosPicker } from './pet-upload-photos-step.hooks';
+import { PhotoPickerProps } from './pet-upload-photos-step.types';
+import imagePickerUtil from '../../../../../shared/image-picker';
+import theme from '../../../../../theme';
+
+const checkForCameraRollPermission = async () => {
+  const status = await imagePickerUtil.getPermissionStatus();
+
+  if (status !== 'granted') alert(i18next.t('PERMISSIONS.CAMERA'));
+  else console.info('Media Permissions are granted');
+};
+
+const PhotoPicker = ({ imageUri, small, onPress, ...props }: PhotoPickerProps) => {
+  const size = small ? 100 : 220;
+
+  return (
+    <Pressable onPress={onPress}>
+      {imageUri ? (
+        <Image rounded="md" source={{ uri: imageUri }} width={size} height={size} {...props} />
+      ) : (
+        <Center
+          rounded="md"
+          backgroundColor={theme.colors.gray[300]}
+          width={size}
+          height={size}
+          {...props}
+        >
+          <Icon name="camera" color={theme.colors.white} size={size / 2.5} />
+        </Center>
+      )}
+    </Pressable>
+  );
+};
+
+const PetUploadPhotosStep = () => {
+  const { images, pickImage } = usePetPhotosPicker();
+
+  useEffect(() => {
+    checkForCameraRollPermission();
+  }, []);
+
+  return (
+    <View>
+      <Center flexDirection="row">
+        <PhotoPicker imageUri={images[0]} onPress={pickImage(0)} alt="Photo 1" testID="Photo 1" />
```

Comment:
```suggestion
        <PhotoPicker imageUri={images[0]} onPress={pickImage(0)} alt="Photo 1" accessibilityHint="uploads the first pet picture" />
```

Please add this text to the locale file